packages/flutter/test/services/hardware_keyboard_test.dart

```diff
@@ -192,7 +193,9 @@ void main() {
       true);
     expect(logs, <int>[3, 2, 1]);
     logs.clear();
-  }, variant: KeySimulatorTransitModeVariant.all());
+
+    focusNode.dispose();
```

Comment:
Please, here and in other tests replace `focusNode.dispose();` with 'addTearDown(focusNode.dispose);', so that if test fails in the middle, it still disposes the object and does not fail with memory leak error.

Reply:
Thanks for the info, I put it directly after the focusNode declaration.